ocs_ci/utility/utils.py

```diff
@@ -1602,6 +1633,27 @@ def email_reports(session):
         log.exception("Sending email with results failed!")
 
 
+def save_reports():
+    """
+    Save reports of test run to slave
+
+    """
+    try:
+        if (
+            "memory" in config.RUN
+            and isinstance(config.RUN["memory"], pd.DataFrame)
+            and not config.RUN["memory"].empty
+        ):
+            stats_dir = create_stats_dir()
+            mem_report_file = os.path.join(stats_dir, "session_mem_report_file")
+            config.RUN["memory"].to_csv(mem_report_file, index=False)
+            log.info(f"Memory performance report saved to '{mem_report_file}'")
```

Comment:
Can we create the documentation for this module under the some section here: https://github.com/red-hat-storage/ocs-ci/tree/master/docs. So the user will know what to set and where they can find the report?

Reply:
done